pgcli/main.py

```diff
@@ -946,14 +949,27 @@ def format_output(title, cur, headers, status, settings):
     case_function = settings.case_function
     formatter = TabularOutputFormatter(format_name=table_format)
 
+    def format_array(val):
+        if val is None:
+            return settings.missingval
+        if not isinstance(val, list):
+            return val
+        return '{' + ','.join(unicode(format_array(e)) for e in val) + '}'
+
+    def format_arrays(data, headers, column_types, **kwargs):
```

Comment:
Since we're not using `column_types` or `kwargs`, it might be more appropriate to make it clear we aren't using any other arguments via the variable name `_`:
```python
def format_arrays(data, headers, **_):
```